apps/frontend/src/app/transformations-creator/transformations-creator.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="container">
+  <div class="row">
+    <div class="col">
+      <form [formGroup]="transformationForm" (ngSubmit)="createTransformation()">
+        <div class="col mb-3">
+          <label for="name" class="form-label">Name:</label><br />
+          <input id="name" class="form-control" type="text" formControlName="name" />
+          <div *ngIf="transformationForm.controls['name'].invalid && transformationForm.controls['name'].touched" class="alert alert-danger">
+            <div *ngIf="transformationForm.controls['name'].errors">
+              Name is required(Max. 40 characters)!
```

Comment:
Nice job with the error message. You don't have to hard code it. Maybe check out how the errors angular give you look with 
```ts
{{transformationForm.controls['name'].errors | json}}
```

Reply:
Used this parameter to get the character limit and actual characters input by the user.